PDF_UA/2/8.9 Annotations/8.9.3 Order/8.9.3.3 Tab order/verapdf-profile-8-9-3-3-t01.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:53:06.894+03:00">
+        <name>ISO 14289-2:202x - 8.9 Annotations - 8.9.3.3 Tab order - Tabs in page with annotations</name>
+        <description>Every page that includes an annotation shall contain a Tabs entry in its page dictionary in
+            accordance with ISO 32000-2:2020, Table 31, and its value shall be A, W or S.</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="PDPage">
+            <id specification="ISO_14289_2" clause="8.9.3.3" testNumber="1"/>
+            <description>Every page that includes an annotation shall contain a Tabs entry in its page dictionary in
+                accordance with ISO 32000-2:2020, Table 31, and its value shall be A, W or S.</description>
+            <test>annots_size == 0 || Tabs == 'A' || Tabs == 'W' || Tabs == 'S'</test>
+            <error>
+                <message>A page containing an annotation contains a Tabs key with value %1 instead of A, W or S.</message>
```

Comment:
A page with annotation(s) contains a Tabs key with value %1 instead of A, W or S.